src/lib.rs

```diff
@@ -994,84 +939,148 @@ mod tests {
     #[cfg(feature = "archive-zip")]
     #[test]
     fn unpack_zip() {
+        test_extract_into(
+            "self_update_unpack_zip_src",
+            "archive.zip",
+            ArchiveKind::Zip,
+        );
+    }
+
+    #[cfg(not(feature = "archive-zip"))]
+    #[test]
+    #[ignore]
+    fn unpack_zip_file() {
```

Comment:
this needed or can it be deleted?

Reply:
This function? The other tests do the same so I just copied them, but happy to remove it.

Reply:
ah right, that's fine then